### ISSUE TITLE
General direction of documentation

### ISSUE THREAD

**Issue by VoyTechnology** [redbrick/docs#24]
While working on updating the documentation I have noticed a few things, mainly lack of clear direction. After #19 is merged, we have essentially accomplished nothing. We still have everything just in somewhat nicer categories. That is why I proposed we move **everything** into legacy. Legacy as in `legacy docs`, not legacy as in `legacy services/procedures`. By doing so we would have:

1. Kept the old documentation in its raw state (or with fixed styling) for reference/lolz.
    - Deleting means loosing precious history of the decisions that were made over time at Redbrick.
2. Make space for brand new docs
    - Probably around 90% of documentation need to be rewritten anyways
    - Not loose old documentation when something is drastically changed. (still in git, but not visible easily)
3. Saved time as 1 and 2 could be worked in parallel.

I do not mind doing all the work, but I can't lead the update process if I do not know what current admins are thinking.

Please join the discussion.

**Comment by butlerx** [comment]
this assumes that everything is unusable 
a lot of the docs are still good just needs to be sorted which #19 does and other ones need be updated with current machine information. archiving and starting from scratch would create more work in the long term.
also documentation can be written in such a way to say how it was done before and why it isn't done that way anymore. or we can just let git history handle that. 
 id prefer to just let git history handle it personally

**Comment by d-fens** [comment]
Delete documentation individually with pull requests where the content is not useful in future projects or re-implementing solutions.

Move outdated items that are useful to legacy. Whether these are displayed is a separate issue.

I would advised that approach as suggested in IRC and here (in different forms)
